micropub.php

```diff
@@ -369,7 +386,7 @@ private static function update() {
 				static::error( 400, 'replace must be an object' );
 			}
 			foreach ( static::mp_to_wp( array( 'properties' => $replace ) )
-					  as $name => $val ) {
+				  as $name => $val ) {
```

Comment:
thanks for fixing this!

Reply:
That first one, the get_users...the sniffer is pretty specific about not letting me adjust that.

Reply:
that's unfortunate, since it's pretty misleading. can you configure it to be more relaxed? often you can disable individual rules in code beautifiers.

i like using phpcs, but it's just a dumb tool, like all tools. i don't want us to be a slave to it when we know better. :P